### DISCUSSION TITLE
How to disable default api ```/ping ```and ```/_upload``` route

### DISCUSSION THREAD

**Discussion by kumaraguru1735** [reflex-dev/reflex#2415] [General]
How to disable default api ```/ping ```and ```/_upload``` route

**Comment by picklelo** [comment]
It's not possible right now. Is there any reason you need to disable them? We can add a config option if it's necessary.

**Comment by kumaraguru1735** [comment]
Yes, We don't want unwanted APIs on my production API

**Reply by ElijahAhianyo** [reply]
The `_upload` route is used internally to perform upload events (send files from the frontend to the backend), and the `ping` route is a convenient route to check the accessibility of the backend server.

**Reply by benedikt-bartscher** [reply]
Could we auto-disable the `_upload` path if no upload component is detected during reflex compilation?  
Edit: I just added a PR which implements this: https://github.com/reflex-dev/reflex/pull/2430
